docs/applications/container-repositories/02-github.md

```diff
@@ -0,0 +1,137 @@
+| `NAMESPACE` | The namespace of your build | variable | If it does not exist paired with the `IMAGE_NAME`, then it will be created for you |
+| `IMAGE_NAME` | The image name of your build | variable | If it does not exist paired with the `NAMESPACE`, then it will be created for you |
+| `REGION` | The region where your cloud container repository is located | variable | Must be a valid cloud region. For example: `eastus` for Azure, or `us-west-1` for AWS. |
+
+## Workflow file
+
+:::note
+
+This GitHub Action will create a new cloud container repository with the following name format: `massdriver<region>`. The region will be the same as the region you set in the `REGION` variable. If the registry already exists, the action will use it.
+
+:::
+
+To set up the GitHub Action, create a new file named `deploy.yaml` in the `.github/workflows` directory of your GitHub repository. You can use this workflow below:
+
+```yaml title=".github/workflows/deploy.yaml"
+name: Deploy to Massdriver
+on:
+  push:
+    branches: [main]
+
+jobs:
+  push_and_deploy:
+    runs-on: ubuntu-latest
+    env:
+      MASSDRIVER_ORG_ID: ${{ secrets.MASSDRIVER_ORG_ID }}
+      MASSDRIVER_API_KEY: ${{ secrets.MASSDRIVER_API_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Massdriver CLI
+        uses: massdriver-cloud/actions@v3
+      - name: Push Image
+        uses: massdriver-cloud/actions/image_push@v3
+        with:
+          namespace: ${{ vars.NAMESPACE }}
+          image-name: ${{ vars.IMAGE_NAME }}
+          artifact: ${{ secrets.AUTHENTICATION_ARTIFACT_ID }}
+          region: ${{ vars.REGION }}
+          image-tag: ${{ github.sha }}
+          build-context: ./
+      - name: Publish Bundle 
+        uses: massdriver-cloud/actions/bundle_publish@v3
+        with:
+          build-directory: ./
+      - name: Set Image Version 
+        uses: massdriver-cloud/actions/app_patch@v3
+        with:
+          project: <insert-project-name-here>
+          target: <insert-environment-name-here>
+          manifest: <insert-bundle/app-manifest-here>
+          set: |
+            <insert-image-tag-path> = "${{ github.sha }}"
+      - name: Deploy App
+        uses: massdriver-cloud/actions/app_deploy@v3
+        with:
+          project: <insert-project-name-here>
+          target: <insert-environment-name-here>
+          manifest: <insert-bundle/app-manifest-here>
+
+```
+
+You may need to modify the branch name in the `on` section of the workflow to match the branch you're using for your application.
+
+When this GitHub Action runs, it will:
+* Build and push your image to your cloud container repository
+* Update the tag in your application package
+* Redeploy your application in Massdriver with the updated tag
+
+### Where can I find my project, target, and manifest names?
```

Comment:
We should add where do i find my massdriver org id and massdriver service account.

Reply:
I added them in the table of secrets and vars here: https://github.com/massdriver-cloud/docs/blob/2bbb318a8a85d9a25f60430807e419ec453cd487/docs/applications/container-repositories/02-github.md#set-secrets-and-vars

Reply:
Added a link to the section above